First attempt to add ergocub sw documentation (#318)
icub-tech-iit/documentation: docs/icub_operating_systems/icubos/jetpack.md

# The NVIDIA JetPack - Introduction

> This guide is intended for a developer that wants to flash and setup the [Nvidia Jetson Xavier boards](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-xavier-series/) (i.e. AXG, NX).

Reviewer: ```suggestion
This guide is intended for developers who want to flash and set up the [Nvidia Jetson Xavier boards](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-xavier-series/) (i.e., AXG, NX).
```
